vulnerabilities/templates/base.html

```diff
@@ -1,53 +1,43 @@
-<!DOCTYPE html class="has-navbar-fixed-top">
 {% load static %}
+<!DOCTYPE html>
+<html lang="en">
+
 <head>
-    <title>VulnerableCode</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% block title %}VulnerableCode.io{% endblock %}</title>
+    <link rel="icon" href="{% static 'images/favicon.ico' %}" />
+
+    <link rel="stylesheet" href="{% static 'css/bulma.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/custom.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/font-awesome.css' %}" />
+    <link rel="stylesheet" href="{% static 'css/bulma-tooltip.css' %}" />
+
+    <script type="text/javascript" src="{% static 'js/jquery-3.6.0.js' %}"></script>
+
+    {% block extrahead %}{% endblock %}
 </head>
-<link rel="stylesheet" href="{% static 'css/bulma.css' %}"/>
-<link rel="stylesheet" href="{% static 'css/custom.css' %}"/>
-<link rel="stylesheet" href="{% static 'css/font-awesome.css' %}"/>
-<meta name="viewport" content="width=device-width, initial-scale=1">
+
 <body class="Site">
-    <nav class="navbar is-light" role="navigation" aria-label="main navigation">
-        <div  class="navbar-menu">
-            <div class="navbar-start">
-                <a class="navbar-item" href="{% url 'home' %}">
-                    Home
-                </a>
-                <a class="navbar-item" href="{% url 'redoc' %}">
-                    API Docs
-                </a>
-                <a class="navbar-item" href="{% url 'package_search' %}">
-                    Packages
-                </a>
-                <a class="navbar-item" href="{% url 'vulnerability_search' %}">
-                    Vulnerabilities
-                </a>
+    <div class="container max-desktop-width is-fullheight">
+        {% block content %}{% endblock %}
+
+        <div style="margin-top: 50px;"></div>
```

Comment:
It would be better to use the available margin CSS classes to ensure consistency in the spacing instead of using arbitrary values. See https://bulma.io/documentation/helpers/spacing-helpers/

Reply:
Done.